default.nix

```diff
@@ -237,6 +237,43 @@ in rec {
     };
   };
 
+  dockerImage = args@{exe, name, version}: let
+    dockerImageSetupScript = nixpkgs.dockerTools.shellScript "dockersetup.sh" ''
+      set -ex
+
+      ${nixpkgs.dockerTools.shadowSetup}
+      echo 'nobody:x:99:99:Nobody:/:/sbin/nologin' >> /etc/passwd
+      echo 'nobody:*:17416:0:99999:7:::'           >> /etc/shadow
+      echo 'nobody:x:99:'                          >> /etc/group
+      echo 'nobody:::'                             >> /etc/gshadow
+
+      mkdir -p    /var/run/backend
+      ln -sft /var/run/backend '${exe}'/*
+      ${nixpkgs.findutils}/bin/find /var/run/backend
+
+      chown 99:99 /var/run/backend
+    '';
+
+  in nixpkgs.dockerTools.buildImage {
```

Comment:
Would be nice to expose these arguments as an attrset so that people could use them as a basis for a customized image.